src/resolvers/channel.ts

```diff
@@ -38,21 +38,52 @@ const resolver: Resolvers = {
       if (!auth) throw new AuthenticationError('User is not signed in');
 
       const { Membership: membershipModel, Channel: channelModel } = models;
-      const { name, type, friendsId } = args.channel;
+      const { name, type, friendIds } = args.channel;
 
-      /**
-       * TODO
-       * 1. Check if there is a membership with only two users with `PRIVATE` type.
-       * 2. Create new channel
-       */
+      if (!friendIds || friendIds.length === 0) throw new Error('friendIds is required');
+
+      const channelMembers = [auth.id, ...friendIds];
       const channel = await channelModel.create({
         name,
         type,
       });
 
+      const membershipData = channelMembers.map((userId) => {
+        return {
+          userId,
+          channelId: channel.id,
+          ...((userId === auth.id || channelMembers.length === 2) && {
+            type: MemberType.Owner,
+          }),
+        };
+      });
+      await membershipModel.bulkCreate(membershipData);
+
       return channel;
     },
   },
+  Channel: {
+    membership: (_, args, { models }): Promise<Membership[]> => {
```

Comment:
As explained above, `memberships` (plural) will be better naming in this case.

Reply:
I agree with that. I applied it. Thanks for your review.